torchinductor/scheduler.py

```diff
@@ -362,6 +362,60 @@ def get_priority(self):
         else:
             return len(self.group) - 1
 
+class FusedNode(SchedulerNode):
+    def __init__(self, scheduler: "Scheduler", snodes):
+        super().__init__(scheduler, snodes[0].node, lambda x: None)
+        node = snodes[0].node
+        (
+            self._sizes,
+            _
+        ) = node.simplify_reorder_and_tile()
+        self.snodes = snodes
+        self.group = snodes[0].group
+        read_writes = self.snodes[0].read_writes
+        for node in snodes[1:]:
+            read_writes = dependencies.merge_read_writes(read_writes, node.read_writes)
+        node_bufs = set([node.get_name() for node in snodes])
+        read_writes.reads = [i for i in read_writes.reads if i.name not in node_bufs]
+        read_writes.writes = [i for i in read_writes.writes if i.name not in node_bufs]
+        self.set_read_writes(read_writes)
+
+    # SchedulerNode overrides
+    def can_remove_buffer(self, check_group):
+        return any([node.can_remove_buffer(check_group) for node in self.snodes])
```

Comment:
This doesn't look right.  We should assert False here, then have the caller iterate over `self.snodes`.